src/js/select2/i18n/ka.js

```diff
@@ -0,0 +1,44 @@
+define(function () {
+  // Georgian
+  return {
+    errorLoading: function () {
+      return 'სია ცარიელია.';
+    },
+    inputTooLong: function (args) {
+      var overChars = args.input.length - args.maximum;
+
+      var message = 'გთხოვთ წაშალოთ ' + overChars + ' სიმბოლო';
+
+      if (overChars != 1) {
+        message += 's';
```

Comment:
I have no idea how Geogian pluralization rules work, so I just want to confirm. Do you really just throw an 's' on the end to pluralize these statements?